lib/upipe-modules/upipe_udp.c

```diff
@@ -572,7 +572,7 @@ int upipe_udp_open_socket(struct upipe *upipe, const char *_uri, int ttl,
         else if (bind_addr.ss.ss_family != AF_UNSPEC) {
     normal_bind:
             if (bind(fd, &bind_addr.so, sockaddr_len) < 0) {
-                upipe_err(upipe, "couldn't bind");
+                upipe_err(upipe, "couldn't bind: %m");
```

Comment:
Hum good catch, yes it needs upipe_err_va.

%m is indeed a GNU extension (Glibc extension according to man printf) but we already use it, upipe_udp.c is even the biggest user.

I am pretty sure it doesn't work on Windows: https://msdn.microsoft.com/en-us/library/hf4y5e3w.aspx and it would print "%m" instead of the erorr string.

Don't know if it works on Darwin